Formula/aptos.rb

```diff
@@ -34,7 +34,7 @@ class Aptos < Formula
 
   def install
     system "#{Formula["rustup-init"].bin}/rustup-init",
-      "-qy", "--no-modify-path", "--default-toolchain", "1.70"
+      "-qy", "--no-modify-path", "--default-toolchain", "1.71"
```

Comment:
this should use homebrew's `rust` instead

Reply:
What version is homebrew's Rust on?  Our codebase is constantly upgrading to the latest rust toolchain, so it has to be up to date, or it may fail.

Reply:
That should be fine. We track the `stable` channel, so currently 1.71: https://github.com/Homebrew/homebrew-core/blob/master/Formula/rust.rb

Reply:
In the future, if `stable` is pointing 1.72, but our toolchain is still pointing to 1.71. This probably would fail?

Reply:
Yes, which would be a reason for brew to check for compatibility with the new version

Reply:
Gotcha!

Noob question - How do I point this to homebrew rust stable version instead of hardcoded `1.71`? Is there some kind of env var?

Reply:
So, if I didn't update my rust, but the CLI was updated, I wouldn't get a CLI update until my Rust was also updated?

Reply:
You can replace `depends_on "rustup-init" => :build` with `depends_on "rustup" => :build`. And then the install block can be:

```ruby
system "cargo", "install", *std_cargo_args(path: "crates/aptos"), "--profile=cli"
```

Overriding `RUSTFLAGS` isn't needed since it's already in your [`.cargo/config.toml`](https://github.com/aptos-labs/aptos-core/blob/aptos-node-v1.5.2/.cargo/config.toml#L16)

Reply:
>  I wouldn't get a CLI update until my Rust was also updated?

Assuming you installed Homebrew in the default prefix (as most users do), you could update the two independently since you would be installing from our pre-built binaries (and not need to install/upgrade rust)

Reply:
Thank you @branchvincent 

The code has been updated with recommended suggestion.

Reply:
Unchecked the brew audit box and revert back to use rustup, because using brew rust is causing the build to fail

Reply:
Unchecked the brew audit box and revert back to use rustup, because using brew rust is causing the build to fail

@branchvincent May I get another look again? All checks seems to be passing